src/main/java/chopchop/logic/commands/HelpCommand.java

```diff
@@ -0,0 +1,23 @@
+package chopchop.logic.commands;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Model;
+
+/**
+ * Format full help instructions for every command for display.
+ */
+public class HelpCommand extends Command {
+
+    public static final String COMMAND_WORD = "help";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows program usage instructions.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String SHOWING_HELP_MESSAGE = "Opened help window.";
+
+    @Override
+    public CommandResult execute(Model model) {
+        return new CommandResult(SHOWING_HELP_MESSAGE, true, false);
```

Comment:
for this kind of boolean parameter, i suggest doing something like this:
```suggestion
        return new CommandResult(SHOWING_HELP_MESSAGE, /* showHelp: */ true, /* exit: */ false);
```
so that it's easier to see what the `true` and `false` mean without going into the definition of the class/function.

Reply:
makes sense!